topics/climate/tutorials/climate-101/tutorial.md

```diff
@@ -354,29 +354,37 @@ At the moment, there are [54 ECVs](https://gcos.wmo.int/en/essential-climate-var
 
 > ### {% icon hands_on %} Hands-on: Essential Climate Variables
 >
+> We will look at the [Water Vapor Essential Climate Variable ](https://gcos.wmo.int/en/essential-climate-variables/surface-vapour/): 
+> *The humidity of air near the surface of the Earth affects the comfort and health of humans, livestock and wildlife, the swarming behaviour of insects and the occurrence of plant disease. The humidity near the surface affects evaporation and the strength of the hydrological and energy cycles. Evaporation from the surface of the earth is the source of water in the atmosphere and so is responsible for important feedbacks in the climate system due to clouds and radiation.* 
```

Comment:
Is there a difference between ` The humidity near the surface` and ` The humidity of air near the surface` ?

Reply:
Well, picky climate scientists would argue that using the term "The humidity near the surface" is imprecise so we better use "the humidity of air near the surface"...

Reply:
I was just confused as you use both term, in two nearby sentences ... maybe you can stick to one?